### ISSUE TITLE
`cfn-lint -t FILE` broken in 1.37.1

### ISSUE THREAD

**Issue by liamdawson** [aws-cloudformation/cfn-lint#4182]
### CloudFormation Lint Version

cfn-lint 1.37.1

### What operating system are you using?

Mac 15.5

### Describe the bug

When using 1.37.1, running `cfn-lint -t FILE` returns usage output instead of checking the template.

### Expected behavior

I expect `cfn-lint` to lint the provided template file when it's passed via `-t`.

### Reproduction template

```sh
$ touch test.yaml
$ cfn-lint -t test.yaml
usage: 
Basic: cfn-lint test.yaml
Ignore a rule: cfn-lint -i E3012 -- test.yaml
...
$ cfn-lint test.yaml 
E1001 'Resources' is a required property
test.yaml:1:1
```
